packages/reactotron-server/src/app/reactotron-app.ts

```diff
@@ -0,0 +1,17 @@
+import { ReactotronApp, TimelineCommandOptions } from "reactotron-core-ui"
+
+class Reactotron implements ReactotronApp {
```

Comment:
I wanted the implementation in server itself so offloaded the interface off to the core-ui package and implement it as a singleton here.